packages/interbit/src/manifest/createManifestTree.js

```diff
@@ -34,8 +34,12 @@ const createManifestTree = (interbitConfig, manifest) => {
     minimumJoinconfig
   )
 
+  const chainIds = mergeJoinAndChildChainIds(joins, chains, manifest)
+
   const manifestEntry = {
+    alias: ROOT_CHAIN_ALIAS,
     chainId,
+    chainIds,
```

Comment:
`chainAlias`, `chainId` and `covenant` are obvious, but `chainIds` and `covenants` less so. What chain IDs and covenants are being referred to? Could we have more descriptive labels? I would think `childChainIds`, `joinedChainIds` and `childChainCovenants` would be more explicit, and `childChainAliases` and `joinedChainAliases` would be really helpful to a reader.

Reply:
I second this. `chainId` and `chainIds` are also too easy to mix up.

Reply:
The chainIds being referred to are all joined and child chain IDs. I am at a loss for a name that isn't something extreme like `childAndJoinedChainIds`. I may just replace the aliases with the chainIDs and forget that chainIds property altogether. 

Going to sleep on it but if you can think of a better name in the meantime lmk :)

Reply:
They are glommed together as a reference map of all the chainIds you may interact with within a covenant so they can be used from anywhere with just an alias. Per your (in person) suggestion I think I will go with referenceChainIds and referenceCovenants